### PR TITLE
#624 - Removing 'excludes' of FindBuging

### PR DESCRIPTION
**Pull request by dalifreire**
Issue #624 - Solving puzzle 574-2964594e in pom.xml.

Fixing and removing the following 'excludes' of FindBugs in the pom.xml file:
- findbugs:org.takes.facets.fork.FkRegex$2 `SIC_INNER_SHOULD_BE_STATIC_ANON`
- findbugs:org.takes.http.FtCLI$3	`SIC_INNER_SHOULD_BE_STATIC_ANON`	
- findbugs:org.takes.http.MainRemote$1 `SIC_INNER_SHOULD_BE_STATIC_ANON`
- findbugs:org.takes.rq.RqMultipart$Base `VA_FORMAT_STRING_USES_NEWLINE`

30 minutes to implement this task. It is not enough because the problem is much bigger and requires more work than the slotted time allows. A new puzzle was created so that can be resolved by other people.

### CODE REVIEW

**Comment by dalifreire** [comment]
@davvd Could you find someone to review it?

**Comment by davvd** [comment]
@dalifreire Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@carlosmiranda review this one, please

**Comment by carlosmiranda** [comment]
@dalifreire looks quite good, thanks!

**Comment by carlosmiranda** [comment]
@rultor merge please

File: src/main/java/org/takes/http/MainRemote.java [hunk 1]
```diff
@@ -163,4 +150,44 @@ private static int port(final File file) throws Exception {
...
+        }
+
+        @Override
+        public void run() {
+            try {
+                this.method.invoke(null, (Object) this.passed);
+            } catch (final InvocationTargetException ex) {
+                throw new IllegalStateException(ex);
+            } catch (final IllegalAccessException ex) {
+                throw new IllegalStateException(ex);
```
Comment (yegor256) [inline src/main/java/org/takes/http/MainRemote.java]: @dalifreire see http://www.yegor256.com/2015/12/01/rethrow-exceptions.html

**Comment by yegor256** [comment]
@dalifreire one comment above

File: src/main/java/org/takes/http/MainRemote.java (hunk 1 above)
Comment (dalifreire) [inline src/main/java/org/takes/http/MainRemote.java]: @yegor256 Done!

**Comment by dalifreire** [comment]
@carlosmiranda @yegor256 Thanks for review. Could you take a look if everything is right now?

File: src/main/java/org/takes/http/MainRemote.java [hunk 2]
```diff
@@ -163,4 +150,56 @@ private static int port(final File file) throws Exception {
...
+                    ex
+                    );
+            } catch (final IllegalAccessException ex) {
+                throw new IllegalStateException(
+                    String.format(
+                        "The visibility of the %s method do not allow access.",
+                        this.method.getName()
+                        ),
+                    ex
+                    );
```
Comment (yegor256) [inline src/main/java/org/takes/http/MainRemote.java]: @dalifreire this indentation looks strange to me :) and a few lines above too

**Comment by yegor256** [comment]
@dalifreire looks good to me, but there is something wrong with indentation, see above

File: src/main/java/org/takes/http/MainRemote.java (hunk 2 above)
Comment (dalifreire) [inline src/main/java/org/takes/http/MainRemote.java]: @yegor256 Fixed, thanks! :)

**Comment by dalifreire** [comment]
@yegor256 Indentation fixed! Thanks! Please, check and merge...

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dalifreire** [comment]
@yegor256 Seems something is wrong with AppVeyor. Could you check, pls?

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by yegor256** [comment]
@dalifreire I can't "check", you have to create a ticket if you think that the project is broken, see http://www.yegor256.com/2015/01/15/how-to-cut-corners.html

**Comment by carlosmiranda** [comment]
@dalifreire any updates here?

**Comment by dalifreire** [comment]
@carlosmiranda I'm waiting for the merge... take a look here: #654

**Comment by carlosmiranda** [comment]
@rultor merge

**Comment by dalifreire** [comment]
@yegor256 ping... @davvd is punishing me for delay.... =)

**Comment by dalifreire** [comment]
@yegor256 Please, could you confirm the merge?

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@ypshenychka please, let us know what do you think about this ticket, according to our [QA rules](http://at.teamed.io/qa.html)

**Comment by davvd** [comment]
@rultor deploy now pls

**Comment by ypshenychka** [comment]
@carlosmiranda 
According to our [QA Rules](http://at.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

No issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed, thanks

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks.

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks for the QA review, we'll work better next time

**Comment by davvd** [comment]
@carlosmiranda 10 mins was added to the account of @ypshenychka (for QA review), in transaction 83433697. Many thanks! **20 mins** were added to your account in Transaction ID `83433716` (task took 745 hours and 25 mins). review comments (c=5) added as a bonus. +20 to your rating, your total score is [+2316](http://www.netbout.com/b/35023?open=rating)

**Pull request merged**